minkindr_gtsam/include/kindr/minimal/common-gtsam.h

```diff
@@ -8,4 +8,63 @@ typedef Eigen::Matrix<double, 3, 3> Jacobian3x3;
 typedef Eigen::Matrix<double, 6, 3> Jacobian6x3;
 typedef Eigen::Matrix<double, 6, 6> Jacobian6x6;
 
+namespace kindr {
+namespace minimal {
+
+template <int N>
+Eigen::Matrix<double, N, 1> vectorScalingImplementation(const Eigen::Matrix<double, N, 1> & v, double alpha, gtsam::OptionalJacobian<N, N> H) {
```

Comment:
What about the derivative wrt. the double? 

Please add that to make this function more general.